src/libs/gltf/lv_gltf_view/lv_gltf.h

```diff
@@ -0,0 +1,351 @@
+/**
+ * @file lv_gltf.h
+ *
+ */
+
+#ifndef LV_GLTF_H
+#define LV_GLTF_H
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "../../../lv_conf_internal.h"
+
+#if LV_USE_GLTF
+
+#include "../../../misc/lv_types.h"
+#include "../../../misc/lv_color.h"
+#include "../lv_gltf_data/lv_gltf_model.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+typedef enum {
+    LV_GLTF_AA_OFF = 0, /** Anti aliasing off*/
+    LV_GLTF_AA_ON = 1,  /** Anti aliasing on*/
+    LV_GLTF_AA_DYNAMIC = 2, /** Anti aliasing on only when frame has no movement*/
+} lv_gltf_antialiasing_mode_t;
```

Comment:
Enum type names and field names should start the same ways. E.g.


```suggestion
typedef enum {
    LV_GLTF_AA_MODE_OFF = 0, /** Anti aliasing off*/
    LV_GLTF_AA_MODE_ON = 1,  /** Anti aliasing on*/
    LV_GLTF_AA_MODE_DYNAMIC = 2, /** Anti aliasing on only when frame has no movement*/
} lv_gltf_aa_mode_t;
```

Reply:
Done 74c5445